BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/SelfDiagnostics/SelfDiagnosticsFileWriter.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing.SelfDiagnostics
+{
+    internal class SelfDiagnosticsFileWriter : ISelfDiagnosticsFileWriter
```

Comment:
I am not sure but checking -- shouldn't this be IDisposable to release file handles.?